src/networking.c

```diff
@@ -46,9 +46,12 @@ size_t zmalloc_size_sds(sds s) {
 size_t getStringObjectSdsUsedMemory(robj *o) {
     redisAssertWithInfo(NULL,o,o->type == REDIS_STRING);
     switch(o->encoding) {
-    case REDIS_ENCODING_RAW: return zmalloc_size_sds(o->ptr);
-    case REDIS_ENCODING_EMBSTR: return sdslen(o->ptr);
-    default: return 0; /* Just integer encoding for now. */
+        case REDIS_ENCODING_RAW:
+            return zmalloc_size_sds(o->ptr);
+        case REDIS_ENCODING_EMBSTR:
+            return sdslen(o->ptr) + sizeof(struct sdshdr) + 1;
+        default:
+            return 0; /* Just integer encoding for now. */
```

Comment:
Thanks. Already fixed including internal fragmentation. No need for this fix anymore.